protocols/delegate/test/Delegate-unit.js

```diff
@@ -17,15 +19,25 @@ contract('Delegate Unit Tests', async accounts => {
   const tradeWallet = accounts[1]
   const notOwner = accounts[2]
   const notTradeWallet = accounts[3]
-  let delegate
-  let mockSwap
-  let snapshotId
-  let swapFunction
   const SIGNER_TOKEN = accounts[9]
   const SENDER_TOKEN = accounts[8]
   const MAX_SENDER_AMOUNT = 12345
   const PRICE_COEF = 4321
   const EXP = 2
+  const MOCK_WETH = accounts[2]
```

Comment:
interesting... why reusing `accounts[2]` and `[3]` when `[4-7]` haven't been used?

Reply:
mistake with copy paste. will fix.